src/react-apps/applications/service-development/src/features/deploy/components/deployPaper.tsx

```diff
@@ -0,0 +1,403 @@
+/* tslint:disable:max-line-length */
+
+import { Grid, Paper, Typography } from '@material-ui/core';
+import { createMuiTheme, createStyles, withStyles } from '@material-ui/core/styles';
+import { TypographyProps } from '@material-ui/core/Typography';
+import classNames from 'classnames';
+import * as React from 'react';
+import AltinnButton from '../../../../../shared/src/components/AltinnButton';
+import AltinnIcon from '../../../../../shared/src/components/AltinnIcon';
+// import AltinnSpinner from '../../../../../shared/src/components/AltinnSpinner';
+import altinnTheme from '../../../../../shared/src/theme/altinnStudioTheme';
+import VersionControlContainer from '../../../../../shared/src/version-control/versionControlHeader';
+
+const theme = createMuiTheme(altinnTheme);
+
+const styles = () => createStyles({
+  paperStyling: {
+    padding: 24,
+    maxWidth: 800,
+  },
+  checkIconPositionFix: {
+    position: 'relative',
+    top: '-5px',
+  },
+  bodyTextStyling: {
+    marginLeft: 5,
+    marginTop: 5,
+  },
+  deployButtonInfoText: {
+    color: theme.altinnPalette.primary.grey,
+  },
+  fontSizeTitle: {
+    fontSize: 20,
+  },
+  link: {
+    borderBottom: '1px solid #0062ba',
+  },
+  listItemTitle: {
+    fontSize: theme.overrides.MuiTypography.body1.fontSize,
+    fontWeight: 500,
+  },
+  paperStyleDeployFailed: {
+    backgroundColor: theme.altinnPalette.primary.redLight,
+  },
+  paperStyleDeploySuccess: {
+    backgroundColor: theme.altinnPalette.primary.greenLight,
+  },
+  paperStyleRepoInSync: {
+    backgroundColor: theme.altinnPalette.primary.greyLight,
+  },
+});
+
+interface IDeployPaperProps {
+  classes: any;
+  cSharpCompiles: boolean;
+  deploymentListFetchStatus: any;
+  localRepoInSyncWithMaster: 'ahead' | 'behind' | 'ready';
+  titleTypographyVariant: TypographyProps['variant'];
+  masterRepoAndDeployInSync: boolean;
+  deploySuccess?: boolean;
+  deployFailedErrorMsg?: any;
+  language: any;
+  env: string;
+}
+
+export const DeployPaper = (props: IDeployPaperProps) => {
+  const { classes, localRepoInSyncWithMaster } = props;
+
+  const constMockCompileFiles = ['firstFile.cs', 'secondFile.cs', 'thirdFile.cs'];
+
+  const renderInSyncText = (param: string) => {
+    switch (param) {
+
+      case 'ready':
+        return (
+          <Typography variant='h2' className={classes.listItemTitle}>
+            Du har delt dine endringer med din organisasjon
```

Comment:
guessing this will be updated with keys, why not just do it

Reply:
I will do it when all texts are in place. There might still be some adjustments.

Reply:
Have you read the PR description? :)

Reply:
yeah, but I'm still annoyed that this is always the last thing you fix. Isn't it easier to make changes to texts when they are in the language file

Reply:
Refactoring code with real text is easier than reading the getLanguage keys.

Reply:
Also language keys does not give any value while presenting the features to designer or other developers. It's the feature with least priority.

Reply:
Language implemented.